lib/py/test/test_thrift_file/TestServer.thrift

```diff
@@ -16,7 +16,11 @@
 # specific language governing permissions and limitations
 # under the License.
 #
- 
+
+struct Message {
+   1: optional string body,
+   2: optional i64 num,
+}
```

Comment:
this change should also be reverted?

Reply:
This struct is used in the test cases here: https://github.com/apache/thrift/pull/2487/files#diff-1364ef1b9a20b93088778203e7f99d70ecb498ba6736e11466979edc229f4600R36